deny-not-main-or-master-revision/env0.rego

```diff
@@ -0,0 +1,3 @@
+package env0
+
+import data.env0
```

Comment:
just a knowledge question what is this import? 🤔

Reply:
@tomer-landesman would like your input here, I copy-pasted that